adapters/oidc/js/src/keycloak.js

```diff
@@ -527,7 +527,15 @@ function Keycloak (config) {
         }
 
         var access = kc.resourceAccess[resource || kc.clientId];
-        return !!access && access.roles.indexOf(role) >= 0;
+        if (!access) {
+            return false;
+        } else if (Array.isArray(role)) {
+            return role.some(function(r) {
+                return access.roles.indexOf(r) >= 0;
+            });
+        } else{
+            return !!access && access.roles.indexOf(role) >= 0;
+        }
```

Comment:
Since all of the branches of this statement terminate (the all `return`). There is no need to use `else if` or `else`.
```suggestion
        if (!access) {
            return false;
        }

        if (Array.isArray(role)) {
            return role.some(function(r) {
                return access.roles.indexOf(r) >= 0;
            });
        }

        return access.roles.indexOf(role) >= 0;
```